src/err/mod.rs

```diff
@@ -559,6 +574,31 @@ mod tests {
         let _ = PyErr::fetch(py);
     }
 
+    #[test]
+    fn err_debug() {
+        let gil = Python::acquire_gil();
+        let py = gil.python();
+        let err = py
+            .run("raise Exception('banana')", None, None)
+            .expect_err("raising should have given us an error");
+        assert!(
```

Comment:
Looks this assertion fails at Python 3.6. Maybe you can just weaken the assertion.

Reply:
👍 modified slightly to check the individual fields and use `#[cfg(Py_3_7)]` on the bit that's different